### ISSUE TITLE
Allow more than one speaker in videos

### ISSUE THREAD

**Issue by Danble** [livingtongues/living-dictionaries#169]
One of our users, Alex Manuel Martinez Aguilera, has told us about his need to add more than one speaker in the videos, since in some videos more than one speaker appears.

@jacob-8, do you think we should restrict this feature to video speakers only or should we consider it for audio files as well?

**Comment by jacob-8** [comment]
> should we consider it for audio files as well?

Yeah, I want to move to where most pieces of data that could ever have multiple (audio, pictures, video, speakers, etc...) are arrays but that takes some careful refactoring. But yes, that is the direction we want to go.

**Comment by AnnaLuisaD** [comment]
ok sounds good. Agreed that moving towards having an array for everything will be great. I added this to our Development Roadmap under "preparing". We should also consider how that will affect the "Export" functionality

**Comment by jacob-8** [comment]
> how that will affect the "Export" functionality

There isn't a great solution because of the flat nature of the spreadsheet. Can either separate speakers by a dividing character or conditionally create more columns for speaker2, speaker3, etc...

**Comment by AnnaLuisaD** [comment]
I think extra columns for speaker2 and speaker 3 is an elegant solution for the export spreadsheet

**Comment by AnnaLuisaD** [comment]
agreed that both audio and video need the ability to have multiple speakers. Diego and Jacob can have their own meeting to flesh this out. spreadsheets are weak at reflecting nested and related data. it's good for me to be aware that all of a sudden you are creating many intricacies and then trying to put them in a flat format. adding them to an import or export spreadsheet will add quite a lot of new columns...
